src/Criteria.php

```diff
@@ -38,9 +38,9 @@ class Criteria
      *
      * @return static
      */
-    public static function create()
+    public static function create(bool $accessRawFieldValues = false): self
```

Comment:
This is a breaking change, you should use `func_get_args()` IMO

https://3v4l.org/Z4ZnM